src/views/Player.vue

```diff
@@ -58,7 +58,13 @@ export default {
       browserErrorHandlingValue: {
         'failsafe_type': 'g-form',
         'failsafe_url': ''
-      }
+      },
+      isSeekBarMoving: true,
+      played: [],
+      submitted: [],
+      revised: [],
+      skipped: [],
+      paused: []
```

Comment:
**isSeekBarMoving** ➡️ To detect the cases where the video does not play (only plays while muted, shitty browsers). If the seek bar does not move for 5 seconds, we set this value to `false`

**played[], paused[]**➡️ These arrays hold the time values where that particular action was taken by the user. Note: the time values are not system time values but `this.player.currentTime`. I think it should be the system time values, right?

**submitted, revised, skipped**➡️ These arrays are initialized with zeros, and the length of the array = number of questions. Whenever an answer is skipped, revised, submitted -> the value for that corresponding question is flipped to 1 in the array.

Reply:
I think `this.player.currentTime` is what we want. What would we do with `system time values`?